src/apps/bm_devkit/sdi12_example/user_code/exo3s_sensor.h

```diff
@@ -0,0 +1,103 @@
+//
+// Created by Uma Arthika Katikapalli on 10/7/24.
+//
+#pragma once
+#include <stdint.h>
+#include "OrderedSeparatorLineParser.h"
+#include "Exo3LineParser.h"
+
+#define sizeBuffer 256
+#define sdiSuccess 0
+#define sdiFail 1
+#define sdiTimeout 2		// timeout condition
+#define sdiParityError 3 	// parity error in reply
+#define sdiReplyError 4		//
+#define breakTimeMin 15		// spec is 12 but a bit extra does no harm
+#define markTimeMin 9		// actually 8.33 but I am not OCD
```

Comment:
Can these be names to include `ms` somewhere in the name?
`breakTimeMin` could be confused with minutes,
maybe like `breakTimeMs` is sufficient enough.